<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1290</width>
    <height>687</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1271</width>
      <height>611</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1290</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="action_new"/>
    <addaction name="action_delete"/>
    <addaction name="separator"/>
    <addaction name="action_close"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>编辑</string>
    </property>
    <addaction name="action_cut"/>
    <addaction name="action_copy"/>
    <addaction name="action_paste"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_new"/>
   <addaction name="separator"/>
   <addaction name="action_close"/>
  </widget>
  <action name="action_cut">
   <property name="text">
    <string>剪切</string>
   </property>
  </action>
  <action name="action_copy">
   <property name="text">
    <string>复制</string>
   </property>
  </action>
  <action name="action_paste">
   <property name="text">
    <string>粘贴</string>
   </property>
  </action>
  <action name="action_new">
   <property name="text">
    <string>新建文件</string>
   </property>
   <property name="toolTip">
    <string>新建文件</string>
   </property>
  </action>
  <action name="action_close">
   <property name="text">
    <string>关闭文件</string>
   </property>
  </action>
  <action name="action_delete">
   <property name="text">
    <string>删除文件</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
